target/linux/ath79/image/gen_kt_img.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
Not sure whether this should go somewhere else, e.g. into /scripts ...

Reply:
And btw: do you need bash?

Reply:
My develeopment environment is Ubuntu, so when I call using `#!/bin/sh,` it calls dash and gets error :
`./gen_kt_img.sh: 8: Syntax error: "(" unexpected`
so I changed it to bash.

Reply:
> Not sure whether this should go somewhere else, e.g. into /scripts ...

In a similar case, for UniElec U7623, script gen_mt7623_emmc_img.sh was located in the target/linux/mediatek/image.
So I put this script this folder...

Reply:
> > And btw: do you need bash?
> 
> My develeopment environment is Ubuntu, so when I call using `#!/bin/sh`,
> it calls `dash` and gets error :
> `./gen_kt_img.sh: 8: Syntax error: "(" unexpected`
> so I changed it to `bash`.

You have to remove the "function" keyword.

Reply:
> > > And btw: do you need bash?
> > 
> > 
> > My develeopment environment is Ubuntu, so when I call using `#!/bin/sh`,
> > it calls `dash` and gets error :
> > `./gen_kt_img.sh: 8: Syntax error: "(" unexpected`
> > so I changed it to `bash`.
> 
> You have to remove the "function" keyword.

Okay, now it works with `#!/bin/sh`. Thanks!!